src/v2/Apps/Fair/Components/ExhibitorsLetterNav.tsx

```diff
@@ -0,0 +1,83 @@
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { Text, Flex, Swiper } from "@artsy/palette"
+import { Media } from "v2/Utils/Responsive"
+import { scrollIntoView } from "v2/Utils/scrollHelpers"
+import { ExhibitorsLetterNav_fair } from "v2/__generated__/ExhibitorsLetterNav_fair.graphql"
+
+const LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ".split("").concat(["0-9"])
+
+interface ExhibitorsLetterNavProps {
+  fair: ExhibitorsLetterNav_fair
+}
+
+export const ExhibitorsLetterNav: React.FC<ExhibitorsLetterNavProps> = ({
+  fair,
+}) => {
+  const letters = fair?.exhibitorsGroupedByName?.map(group => group?.letter)
+
+  const Letters = props => {
+    return (
+      <Flex justifyContent="space-between" {...props}>
+        {LETTERS.map(letter => {
+          const isEnabled = letters?.includes(letter)
+          return (
+            <Letter
+              key={letter}
+              color={isEnabled ? "black100" : "black10"}
+              title={
+                isEnabled ? `View exhibitors starting with “${letter}”` : ""
+              }
+              onClick={() => {
+                if (isEnabled) {
+                  scrollIntoView({
+                    selector: `#jump--letter${letter}`,
+                    offset: 120,
+                    behavior: "smooth",
+                  })
+                }
+              }}
+            >
+              {letter}
+            </Letter>
+          )
+        })}
+      </Flex>
+    )
+  }
+
+  return (
+    <>
+      <Media lessThan="md">
+        <Swiper snap="start">
+          <Letters width={1300} />
+        </Swiper>
+      </Media>
+
+      <Media greaterThanOrEqual="md">
+        <Letters />
+      </Media>
+    </>
+  )
+}
+
+const Letter = styled(Text).attrs({ p: 1, variant: "md" })`
```

Comment:
@anastasiapyzhik I think we should remove this padding around each letter - in the specs, the letters are all the way to the left side of the screen and they line up perfectly with the exhibitors. Because they have padding now, they are slightly toward the middle of the screen and look slightly strange.

Reply:
👍  That makes sense for desktop but it seems that we should keep the padding on mobile

![Screenshot 2021-08-23 at 15 55 45](https://user-images.githubusercontent.com/44819355/130450737-f629f639-43a2-4045-a63c-b70f50c42c86.png)